integration-tests/miscellaneous/utilityBatch.test.ts

```diff
@@ -0,0 +1,142 @@
+import assert from "assert";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { ExtrinsicHelper } from "../scaffolding/extrinsicHelpers";
+import { createAndFundKeypair } from "../scaffolding/helpers";
+import { ApiTypes, SubmittableExtrinsic } from "@polkadot/api/types";
+
+describe("Utility Batch Filtering", function () {
+    let sender: KeyringPair;
+    let recipient: KeyringPair;
+
+    before(async function () {
+        sender = await createAndFundKeypair();
+        recipient = await createAndFundKeypair();
+    });
+
+    it("should successfully execute ✅ batch with allowed calls", async function () {
+        // good batch: with only allowed calls
+        const goodBatch: SubmittableExtrinsic<ApiTypes>[] = [];
+        goodBatch.push(ExtrinsicHelper.api.tx.balances.transfer(recipient.address, 1000))
+        goodBatch.push(ExtrinsicHelper.api.tx.system.remark("Hello From Batch"))
+        goodBatch.push(ExtrinsicHelper.api.tx.msa.create())
```

Comment:
Non-blocking:  What's the reasoning for allowing Msa::create but disallowing create_provider?